### PR TITLE
feat:  bring the latest icon library

### LINKED ISSUE #1386: Fiori 3.0 Visual Update: Update Icon Library
We are missing some icons in the fundamentals library compared to what is available on Sap icon library (32 to be exact). 

> Side note: we are specifically looking for the `task` icon to use in Smartedit/Personalization 
Screen Shot 2019-04-12 at 8 40 14 AM

[Sap Icon Library](https://sapui5.hana.ondemand.com/test-resources/sap/m/demokit/iconExplorer/webapp/index.html#/overview/SAP-icons/?tab=grid
) - switch to "Sap Icons" from the top bar 

Todo: 
- add new missing icons to the fundamentals library 
- update downloadable icons set on the UI Kit page

Attached is the latest set of icon font files 
[Icon Fonts.zip](https://github.com/SAP/fundamental/files/3073475/Icon.Fonts.zip)

### PR DESCRIPTION
**Pull request by droshev**
fixes #1386

### CODE REVIEW

File: docs/_data/icons.yml [hunk 1]
```diff
@@ -354,6 +354,8 @@
   unicode: "\e0b0"
 - name: message-error
   unicode: "\e0b1"
+- name: notification
+  unicode: "\e0b1"
 - name: comment
```
Comment (bcullman) [inline docs/_data/icons.yml]: there are some errors in this file. you cannot have 2 icons occupy the same unicode value. This needs to be redone.

**Comment by eboyer** [comment]
@droshev ~ I believe @saad-mo had been working on this story as well. Maybe we could follow up with a quick fix for what @bcullman mentioned and then check in with @saad-mo for a review to compare against anything he may of had in progress?

File: docs/_data/icons.yml (hunk 1 above)
Comment (droshev) [reply docs/_data/icons.yml]: The source of this is the sap icons. I didn’t modify any name. I guess we need to address this to the team generating the names. There was another mistake i found-the last icon in the icon library is named collapse-all, even though there is another one

Comment (droshev) [reply docs/_data/icons.yml]: I will remove the duplicate for now and maybe we can try to automate the process of including the icon library

Comment (bcullman) [reply docs/_data/icons.yml]: it is *possible* that they are using 2 names for the same icon. 🤷‍♂ 

I would check with them before removing anything.

Comment (droshev) [reply docs/_data/icons.yml]: in their css file is written `duplicate icon with alternative class name for compatibility reasons`

**Comment by mikeRujevic** [comment]
Hi @droshev, can we add all icons from SAP-icons-extended font family ?

**Pull request merged**